Add a note regarding the class mediator property (#2603)
wso2/docs-ei: en/micro-integrator/docs/references/mediators/class-Mediator.md

!!! Info
    If you want, you can pass the same variables as a value or an expression:

    -   Example for passing the variable as a value: `          <property name="variable1" value="10"/>         `

    -   Example for passing the variable as an expression: `          <property name="variable2" expression="get-property('variable1')"/>         `  
        For more information on using the get property method, see the [Property Mediator](property-Mediator.md).

!!! Note

>         Using the class variables with expression will lead to concurrency issues.

Reviewer: ```suggestion
        Using the class variables with expression will lead to the values evaluated being mixed up when there are are concurrency requests and will lead to erroneous behaviors. 
```

Also shall we provide an alternative for it, that is to pass the expression as string and evaluate in mediate method

>         Using the class variables with expression will lead to the values evaluated being mixed up when there are concurrent requests and will lead to erroneous behaviors. 

Reviewer 2: ```suggestion
        Using the class variables with expressions will lead to the values evaluated being mixed up when there are concurrent requests and will lead to erroneous behaviors. 
```

>         Using the class variables with expression will lead to the values evaluated being mixed up when there are are concurrency requests and will lead to erroneous behaviors. 

Reviewer: ```suggestion
        Using the class variables with expression will lead to the values evaluated being mixed up when there are are concurrent requests and will lead to erroneous behaviors. 
```
